cms/release_calendar/models.py

```diff
@@ -294,3 +297,12 @@ def get_lock(self) -> Optional["BaseLock"]:
             return PageInBundleReadyToBePublishedLock(self)
 
         return super().get_lock()
+
+    @cached_property
+    def cached_analytics_values(self) -> dict[str, str | bool]:
```

Comment:
This gets repeated, should we consider a mixin?

Reply:
Although a few implementations look very similar, they're not quite identical because the field name for `lastUpdatedDate` varies between models, which I think makes it more trouble than it's worth to try and factor the common shape out into a mixin.